Migration produces an error due to lack of migrations

How to reproduce:
- Run 0.1.0
- Do backup for update
- Run 0.1.1-rc1

```
[ERROR] The version "latest" couldn't be reached, there are no registered migrations.
```

Solution:
Add empty migration.

Cannot use:
```
php bin/console doctrine:migrations:migrate --allow-no-migration
```
still produces a warning.

```
[WARNING] The version "latest" couldn't be reached, there are no registered migrations.
```

--- a/config/migrations/DoctrineMigrationTemplate.php.template
+++ b/config/migrations/DoctrineMigrationTemplate.php.template
@@ -21,24 +21,26 @@ use Doctrine\Migrations\AbstractMigration;
 /**
  * ! Migration file MUST **NOT** BE dependent on application code.
  *
- * @see devDocs/Doctrine migrations.md for more details and example queries
+ * @see development/Doctrine migrations.md for more details and example queries
  *
- * Migration for version vTODO
- * Migration for task TODO Functionality name
+ * Migration from version TODO
+ * Migration for issue #TODO_ISSUE_NUMBER Issue title
  */
 final class <className> extends AbstractMigration
 {
     public function preUp(Schema $schema): void
     {
+        // TODO this preUp() migration is auto-generated, modify it to your needs and remove this comment. When not used, remove the this function
     }
 
     public function up(Schema $schema): void
     {
-        // TODO this up() migration is auto-generated, please modify it to your needs and remove this comment
+        // TODO this up() migration is auto-generated, modify it to your needs and remove this comment
 <up>
     }<override>
 
     public function postUp(Schema $schema): void
     {
+        // TODO this postUp() migration is auto-generated, modify it to your needs and remove this comment. When not used, remove the this function
     }
 }

--- a/development/Doctrine migrations.md
+++ b/development/Doctrine migrations.md
@@ -1,0 +1,87 @@
+# Doctrine migrations
+
+## Design assumptions
+
+-   Doctrine migration file should be created for each issue when database change or files change is required
+-   `preUp` and `postUp` methods should be used to handle database content updates and files changes
+-   `up` method is used to change database schema
+-   Migration file MUST **NOT** BE dependent on application code
+    -   because if application code will change it might influence migration file execution and this will mean additional maintenance work
+    -   needed code should be copied into migration file
+    -   general packages like `Arr`, `Symfony Filesystem`, etc. can be used, but keep in mind that using them will require additional maintenance when upgrading packages
+    -   usage of container should be limited to `Symfony parameters`, and used with caution
+
+## General comments
+
+Default migration template is overridden by `config/migrations/DoctrineMigrationTemplate.php.template`. This template includes additional reminder in comments and information about which task is connected to this migration. Please fill `#TODO_ISSUE_NUMBER Issue title` with a task number and a brief functionality name (i.e. `#13 Migration produces an error due to lack of migrations`) and fill `version vTODO` with a version number. We do not support nor implement `down()` function.
+
+## Generating migration
+
+Use following command:
+
+```
+php bin/console doctrine:migrations:diff
+```
+
+## Doctrine migrations limitations
+
+-   Doctrine migrations are executed in sequence (preUp(), up(), postUp() -> next migration file) until exception occurs
+-   When exception occurs changes in database are to be rollback, however
+    -   **!IMPORTANT!** when any of schema change SQL is executed previous changes are automatically committed, eg. CREATE TABLE, ALTER TABLE, DROP TABLE, etc.
+-   Doctrine migration code is executed as written in methods, except in up() method each addSql() method is just collecting SQL to be executed when up method is finished
+    -   **!IMPORTANT!** $this->addSql() line DOES NOT EXECUTE SQL during up() method execution
+-   Any $this->connection-> sql execution, executes immediately
+
+## Example SQL statements
+
+For details see vendor/doctrine/dbal/src/Connection.php
+
+Many ways to use UPDATE (INSERT is similar)
+
+```
+$this->connection->update('user', ['extra_field' => 'SecretValue'], ['username' => 'system']);
+$this->connection->executeStatement('UPDATE user SET `extra_field`= "SecretValue" WHERE `username` = "system"');
+$this->connection->executeStatement('UPDATE user SET `extra_field`= ? WHERE `username` = ?', ['SecretValue', 'system']);
+```
+
+Getting lastInsertId
+
+```
+$deviceTypeId = $this->connection->fetchOne('SELECT id FROM device_type WHERE `name`= ?', ['TK800']);
+if (!$deviceTypeId) {
+    throw new InvalidDatabaseStateException('deviceType TK800 not found');
+}
+
+$this->connection->executeStatement('INSERT INTO template (`device_type_id`, `name`) VALUES (?,?)', [$deviceTypeId, 'template1']);
+$templateId = $this->connection->lastInsertId();
+if (!$templateId) {
+    throw new InvalidDatabaseStateException('Last insert ID not found');
+}
+
+$this->connection->insert('template_version', ['template_id' => $templateId, 'device_type_id' => $deviceTypeId, 'name' => 'templateVersion1', 'type' => 'staging']);
+```
+
+Safely inserting records (with check if already exists)
+
+```
+$tk500v3Count = $this->connection->fetchOne(
+    'SELECT count(*) FROM device_type WHERE `name` = ? OR `slug` = ? OR `route_prefix` = ?',
+    ['TK500v3', 'tk500v3', '/router/tk500-v3']
+);
+
+if ($tk500v3Count > 0) {
+    // TK500v3 already exists do nothing
+    return;
+}
+
+//This code is just example (not valid amount of properties)
+$this->connection->insert('device_type', [
+    'name' => 'TK500v3',
+    'icon' => 'router',
+    'device_name' => 'Router',
+    'color' => '#AF0AE1',
+    'slug' => 'tk500v3',
+    'certificate_common_name_prefix' => 'tk5003',
+    'enabled' => 1,
+]);
+```

--- a/migrations/Version20251215071454.php
+++ b/migrations/Version20251215071454.php
@@ -1,0 +1,36 @@
+<?php
+
+// Copyright (c) 2025 Contributors to the Eclipse Foundation.
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache License, Version 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// SPDX-License-Identifier: Apache-2.0
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * ! Migration file MUST **NOT** BE dependent on application code.
+ *
+ * @see development/Doctrine migrations.md for more details and example queries
+ *
+ * Migration from version 0.1.0
+ * Migration for issue #13 Migration produces an error due to lack of migrations
+ */
+final class Version20251215071454 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        // Fake SQL to avoid migration reporting an error
+        $this->addSql('SELECT 1');
+    }
+}

